src/frontend/components/SøknadsSteg/Dokumentasjon/filopplaster/OpplastedeFiler.tsx

```diff
@@ -33,9 +33,10 @@ const FilTekstWrapper = styled.div`
     margin-right: 1rem;
 `;
 
-const StyledAttachment = styled(Attachment)`
+const StyledAttachment = styled(PaperclipIcon)`
     margin-right: 1rem;
     min-width: 1rem;
+    font-size: 1.3rem;
```

Comment:
Ble denne for stor med 1.5rem? 🤔

Reply:
Fikset større ikon nå:
![image](https://github.com/navikt/familie-ba-soknad/assets/25459913/b7b923d4-139c-4bb9-8ac9-bc1639369f6d)